velox/dwio/parquet/tests/ParquetTableScanTest.cpp

```diff
@@ -86,3 +86,25 @@ TEST_F(ParquetTableScanTest, basic) {
 
   assertQuery(plan, {split}, "SELECT min(a), max(b) FROM tmp");
 }
+
+TEST_F(ParquetTableScanTest, emptyProjection) {
```

Comment:
I will update this test based on https://github.com/facebookincubator/velox/pull/873/